.lando.yml

```diff
@@ -147,7 +147,7 @@ services:
   # This service adds elasticsearch, used by (you guessed it) the search part of
   # the application.
   elasticsearch:
-    type: compose
+    type: lando
```

Comment:
Apparently in new versions of lando the `compose` service is [now deprecated](https://docs.lando.dev/plugins/compose/) and the `lando service` should be used instead.